app/views/users/show.html.erb

```diff
@@ -2,16 +2,17 @@
   <!-- pseudoflash because putting it in a real one was making for some ugly code -->
   <div class="flash notice" id="first-login-help-banner">
     <p>
-      Hi! It looks like you've just logged into the archive for the first time. 
-      <%= link_to_modal ts("Learn some tips and tricks"), :for => first_login_help_path, :title => "First login help" %> 
-      or dismiss this message permanently.
+      <%= t(".login_banner.welcome_text", help_link: link_to_modal(t(".login_banner.help_link"), for: first_login_help_path, title: t(".login_banner.help_title"))).html_safe %>
     </p>
-      <%= form_tag end_first_login_user_path(current_user), :method => :post, :remote => true do %>
-    <p class="submit actions">
-      <%= submit_tag ts("Dismiss permanently") %>
-      <%= link_to "x", nil, :remote => true, :id => "hide-first-login-help", :title => "Hide first login help banner" %>
+    <p>
+      <%= t(".login_banner.abuse_text", abuse_link: link_to(t(".login_banner.abuse_link"), new_abuse_report_path)).html_safe %>
     </p>
-      <% end %>
+    <%= form_tag end_first_login_user_path(current_user), method: :post, remote: true do %>
+      <p class="submit actions">
+        <%= submit_tag t(".login_banner.dismiss") %>
+        <%= link_to "&times;".html_safe, nil, remote: true, id: "hide-first-login-help", title: t(".login_banner.hide") %>
```

Comment:
It looks like lines 42 and 49 of features/other_a/banner_login.feature need to be updated to use the new character.